src/main/groovy/de/dkfz/roddy/execution/jobs/BEJob.groovy

```diff
@@ -130,12 +124,30 @@ class BEJob<J extends BEJob, JR extends BEJobResult> {
         this.toolMD5 = toolMD5
         this.resourceSet = resourceSet
         this.parameters = parameters
-        this.parentJobs = parentJobs
-        this.arrayIndices = arrayIndices ?: new LinkedList<String>()
-        this.listOfCustomDependencyIDs.addAll(parentJobIDs ? parentJobIDs : new LinkedList<BEJobID>())
         this.jobManager = jobManager
     }
 
+    static BEJob fromJobID(BEJobID jobID, BatchEuphoriaJobManager jobManager = jobID?.job?.jobManager) {
```

Comment:
I'll change that into a constructor.

No reference to an ID from job, yet. Gonna change.